Libraries/Opc.Ua.Configuration/ApplicationConfigurationBuilder.cs

```diff
@@ -104,7 +104,9 @@ public IApplicationConfigurationBuilderClientSelected AsClient()
             var rejectedRootType = CertificateStoreIdentifier.DetermineStoreType(rejectedRoot);
             ApplicationConfiguration.SecurityConfiguration = new SecurityConfiguration {
                 // app cert store
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
there is code in the field that directly tinkers with ApplicationCertificate, we need to make sure it breaks.